src/pages/worklist/DOIRequestList.tsx

```diff
@@ -1,15 +1,74 @@
-import React from 'react';
-import { Typography } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { CircularProgress, Typography } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
+import { getWorkListItemDOI } from '../../api/workListApi';
+import ErrorBanner from '../../components/ErrorBanner';
+import { WorklistDOIRequest } from '../../types/Worklist.types';
+import DOIRequestItem from './DOIRequestItem';
+import { getResource } from '../../api/resourceApi';
+import { Resource } from '../../types/resource.types';
+import { AxiosResponse } from 'axios';
+import styled from 'styled-components';
+
+const StyledUl = styled.ul`
+  list-style: none; /* Remove list bullets */
+  padding: 0;
+  margin: 0;
+`;
 
 const DOIRequestList = () => {
   const { t } = useTranslation();
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState<Error>();
+  const [workListDoi, setWorkListDoi] = useState<WorklistDOIRequest[]>([]);
+
+  useEffect(() => {
+    const fetchWorkLisDoi = async () => {
```

Comment:
typo in fetchWorkLisDoi.  + DOI med store bokstaver - siden vi har det andre steder

Reply:
Fikset